src/resty/aws/credentials/RemoteCredentials.lua

```diff
@@ -76,6 +79,16 @@ local function initialize()
                     ({ http = 80, https = 443 })[FullUri.scheme]
   end
 
+  -- get auth token
+  if aws_config.global.AWS_CONTAINER_AUTHORIZATION_TOKEN then
+    AuthToken = aws_config.global.AWS_CONTAINER_AUTHORIZATION_TOKEN
+  end
+
+  -- get auth token file path
+  if aws_config.global.AWS_CONTAINER_AUTHORIZATION_TOKEN_FILE then
+    AuthTokenFile = aws_config.global.AWS_CONTAINER_AUTHORIZATION_TOKEN_FILE
+  end
+
```

Comment:
you don't need the if statements here
```suggestion
  -- get auth token/file
  AuthToken = aws_config.global.AWS_CONTAINER_AUTHORIZATION_TOKEN
  AuthTokenFile = aws_config.global.AWS_CONTAINER_AUTHORIZATION_TOKEN_FILE

```

Reply:
was this lost in the rebase?

Reply:
ah yes, good catch

Reply:
updated, added back the suggestion fix